Microwave.App/Program.cs

```diff
@@ -30,7 +31,7 @@ static void Main(string[] args)
 
             CookController cooker = new CookController(timer, display, powerTube);
 
-            UserInterface ui = new UserInterface(powerButton, timeButton, startCancelButton, door, display, light, buzzer, cooker);
+            UserInterface ui = new UserInterface(powerButton, timeButton, startCancelButton, subtractTimeButton, door, display, light, cooker);
```

Comment:
BuzzerFeature still needs to be a parameter in the UserInterface constructor.

Reply:
Has been added.